data/human/deep missions.txt

```diff
@@ -2431,3 +2431,340 @@ event "battle in arneb end"
+			`	The woman, who you've learnt is named Ragnhild, introduces Thorleif to some of the folks here and before long they have found ways to connect their ancient genealogies together and everyone is suddenly treated as family. You eat their meager fare, swap stories, and listen to legends about the old times.`
+			`	As the sun begins to set Thorleif approaches you, "It is time," he says. "The journey to the cairn is only a short hike from here. I would that you accompany me. I think Skaldgar would have wanted that."`	
+			choice
+				`	(Leave Grimssalr with Thorleif.)`
+					goto grimfollow
+			label grimfollow
+			`	You follow Thorleif up a mostly overgrown trail for about 30 minutes before finally stepping out into a wide open plain. The wind is cool and refreshing and the grass blows with a steadiness that is somehow reassuring. There is something special, timeless, about this sudden meadow.`
+			`	You notice over two dozen columns of lain stone arranged in a circle around a much more massive and impressive structure. The lain stone columns must be cairns of various nearby families, but the massive structure in the center is something completely different. The structure is a wall, cut from some sort of metal you don't recognize and filled with letters and images that are unfamiliar to you.`
+			`	"That is an Elf-Stone. It is why we built our Family-Cairns here. Such stones can be found on most worlds in The Deep. Indestructible, and left by the Alfar for some unknowable reason. Some, like this one, have been mostly forgotten. Others though, are important places of celebration and even worship.`
+			`	He points at a smallish cairn next to the Elf-Stone. "This, though, is what we're here for."`
+			choice
+				`	(You eye the worn and weather beaten cairn with a faded name plate reading, 'Hlewagast' carved at its base.)`
+					goto thorcarn
+			label thorcarn
+			`	Thorleif approaches the cairn.`
+			`	Thorleif holds up the stone of his father in his palm and begins to read the runes carved upon it.`
+			`	"The day my mother gave me this stone and said she was proud of me."`
+			`	"The day my son Thorleif was born."`
+			`	"The day I struck the mother lode on Helheim."`
+			`	"The day I met <first> and he helped me find my family again."`
+			`	"The day I saw my son again, even though he would not see me."`
+			`	Thorleif sniffs, and takes a moment to write something down he saw on the stone, and then walks forward and places it upon the cairn, at home now with hundreds of other stones each with their own handful of memories.`
+			`	"He had not so many good memories, I think," Thorleif said quietly. "I realize now, that mine too are quite few. Perhaps that, if nothing else, is a lesson he can teach me. For that, I thank you, dad."`
+			`	With that Thorleif turns from the cairn with blurry eyes. "It is done. Let us head back."`
+			choice
+				`	(Follow Thorleif back to the village.)`	
```

Comment:
This would be a spot where I'd say keep it in. This is a good pause since we're going back to the village.